Pipeline/PackageBuildOutputs/README.md

```diff
@@ -566,9 +569,23 @@ As an example, you can invoke the SBOM generation with the following command:
 /usr/lpp/dbb/v2r0/bin/groovyz -cp /u/mdalbin/SBOM/cyclonedx-core-java-8.0.3.jar:/u/mdalbin/SBOM/jackson-annotations-2.16.1.jar:/u/mdalbin/SBOM/jackson-core-2.16.1.jar:/u/mdalbin/SBOM/jackson-databind-2.16.1.jar:/u/mdalbin/SBOM/jackson-dataformat-xml-2.16.1.jar:/u/mdalbin/SBOM/json-schema-validator-1.2.0.jar:/u/mdalbin/SBOM/packageurl-java-1.5.0.jar /u/mdalbin/SBOM/dbb/Pipeline/PackageBuildOutputs/PackageBuildOutputsWithSBOM.groovy --workDir /u/ado/workspace/MortgageApplication/feature/consumeRetirementCalculatorServiceImpacts/build-20240312.1/logs --tarFileName MortgageApplication.tar --addExtension -s -sa "David Gilmour <david.gilmour@pinkfloyd.com>"
 ~~~~ 
 
-By default, the SBOM file is generated in the `tempPackageDir` and named `sbom.json`.
+By default, the SBOM file is generated in the `tempPackageDir` and named `<buildnumber>_sbom.json`.
 This way, it is automatically packaged in the TAR file that is created by the script, ensuring the package and its content are not tampered and correctly documented. 
 
+## IBM Concert Build (SBOM) manifest generation
```

Comment:
It's not really an SBOM file, it's more an IBM Concert Configuration file to me. Could we rename that section and following references to IBM Concert SBOM file to something else, to avoid confusion?

```suggestion
## IBM Concert Build manifest generation
```